.github/workflows/release.yml

```
@@ -0,0 +1,62 @@
+name: release
+on:
+  push:
+    branches: [master, main]
+    tags: ["*"]
+    paths-ignore: ['**.md']
+  workflow_dispatch: #allows manual trigger
+
+concurrency: production
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: '0'
+      - name: Set up JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '21'
+      - name: Get next release version (dry run)
+        id: taggerDryRun
+        uses: anothrNick/github-tag-action@1.61.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          WITH_V: true
+          DRY_RUN: true
+      - name: echo new tag
+        run: |
+          export NEXT_VERSION=${{ steps.taggerDryRun.outputs.new_tag }}
+          echo "The next tag version will be: $NEXT_VERSION"
+      - name: echo tag
+        run: |
+          echo "The current tag is: ${{ steps.taggerDryRun.outputs.tag }}"
+      - name: Compile
+        run: sbt clean stage
+      - name: Generate Ruby Types
+        run: ./standalone --withLanguageFrontend RUBYSRC 
+      - name: Checksum
+        run: sha512sum builtin_types/ruby_builtin_types.zip > builtin_types/ruby_builtin_types.zip.sha512
+      - name: Rename
+        run: |
+          mv builtin_types/ruby_builtin_types.zip ruby_builtin_types.zip
```

Could we make this match the name of the frontend? Then we can use `generated.Languages.RUBYSRC` to help build the URL to download this from
```suggestion
          mv builtin_types/ruby_builtin_types.zip rubysrc_builtin_types.zip
```